validator/app.py

```diff
@@ -23,6 +26,14 @@
 logger = logging.getLogger(__name__)
 app = FastAPI()
 
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=["*"],  # Allows all origins
+    allow_credentials=True,
+    allow_methods=["*"],  # Allows all methods
+    allow_headers=["*"],  # Allows all headers
+)
+
```

Comment:
I dont know context but at first glance, cors should be pin to specified headers, methods, and origins due to security

Reply:
This is to enable Lukasz to connect, if he succeeded, then we can think how to make it restricted, i dont know if origin should be client or gateway or maybe even both

Reply:
okay got it, when we switched to validator as a service? Iirc it was abanodned when validation started to work

Reply:
We haven't removed it yet

Reply:
Connect from where? local dev env to local dev env. or from local client dev to test instance?

Reply:
@kacper516 imo you could remove all this directory - since we dont event use it